test/sslapitest.c

```diff
@@ -7641,6 +7744,7 @@ int setup_tests(void)
     ADD_ALL_TESTS(test_early_data_skip_abort, 3);
     ADD_ALL_TESTS(test_early_data_not_sent, 3);
     ADD_ALL_TESTS(test_early_data_psk, 8);
+    ADD_ALL_TESTS(test_early_data_psk_with_all_5_ciphers, 5);
```

Comment:
This name seems unlikely to age well.

Reply:
Can I rename it as `test_early_data_psk_with_rfc8446_ciphersuites()` ?

Reply:
I'd just drop the `5` from the name, e.g. `test_early_data_psk_with_all_ciphers`.

Reply:
Changes done.